product_pricelist_supplierinfo/models/product_pricelist.py

```diff
@@ -44,3 +44,6 @@ class ProductPricelistItem(models.Model):
     no_supplierinfo_min_quantity = fields.Boolean(
         string='Ignore Supplier Info Min. Quantity',
     )
+    filter_supplier_id = fields.Many2one(
```

Comment:
Does it makes sense to make this a many2many for being more versatile?

Reply:
Hi @pedrobaeza , you can achieve the same goal by creating one rule for every supplier. I think it's more clear for end user.

Our use case is: for all the products supplied by a vendor, apply markup.
With a m2m, what's the meaning of the filter when you have more vendors for the same product?

Reply:
OK